A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -73,6 +73,9 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 
         this.electricity = this.querySelector("#Electricity");
         this.climbTransitionGroundAltitude = null;
+        this.initB = false;
+        this.initBTimer = 0;
```

Comment:
Looks like this isn't used anywhere?

Reply:
Good catch, that's a left over I forgot to clean up

Reply:
Removed.